LuaUI/Widgets/cmd_mex_placement.lua

```diff
@@ -650,6 +679,25 @@ function widget:Update()
 		end
 	end
 
+	if debounceCamUpdate then
+		debounceCamUpdate = debounceCamUpdate - dt
+		if debounceCamUpdate < 0 then
+			debounceCamUpdate = nil
+		end
+	else
+		local cx, cy, cz = Spring.GetCameraDirection()
+		local newCamDir = ((math.atan2(cx, cz) / math.pi) + 1) * 180
+		if newCamDir ~= camDir then
+			camDir = newCamDir
+			gl.DeleteList(incomeLabelList)
+			incomeLabelList = glCreateList(DrawIncomeLabels)
+			debounceCamUpdate = .1
```

Comment:
Github's syntax highlighter doesn't color this like a number and I'm scared (also this style is not used anywhere else)
```suggestion
			debounceCamUpdate = 0.1
```